functions/functionsv2/label_gce_instance/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/GoogleCloudPlatform/golang-samples/functions/functionsv2/label_gce_instance
+
+go 1.17
+
+require github.com/cloudevents/sdk-go/v2 v2.6.1
+
+require (
+	cloud.google.com/go v0.97.0
+	github.com/googleapis/gax-go/v2 v2.1.0
+	github.com/json-iterator/go v1.1.10 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180228061459-e0a39a4cb421 // indirect
+	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
+	google.golang.org/genproto v0.0.0-20210924002016-3dee208752a0
+	google.golang.org/protobuf v1.27.1
+)
+
+require (
```

Comment:
Not familiar with this syntax using multiple require blocks. Perhaps a `go mod tidy` is in order?

Reply:
This was not an intentional choice, and tidy does not merge the two blocks.
I've merged them manually and everything seems fine.